ansible/config_sonic_basedon_testbed.yml

```diff
@@ -74,6 +74,101 @@
         vm_base: "{% if testbed_facts['vm_base'] != '' %}{{ testbed_facts['vm_base'] }}{% else %}''{% endif %}"
     when: testbed_name is defined
 
+  - name: Run L1 Switch connection panel for RDMA testing
+    block:
+    - name: Get HWSKU type
+      set_fact:
+        hwsku_type: "{{ hostvars[inventory_hostname]['hwsku'] }}"
+
+    - name: Create connections for 8111 testbed
+      block:
+      - name: Create 1st port connection for Cisco 8111 testbed
+        raw: connection create p40 to p45
+        ignore_errors: true
+      
+      - name: Pause for 2 seconds
+        pause:
+          seconds: 2
+      
+      - name: Create 2nd port connection for Cisco 8111 testbed
+        raw: connection create p41 to p46
+        ignore_errors: yes
+      
+      - name: Pause for 2 seconds
+        pause:
+          seconds: 2
+      
+      - name: Create 3rd port connection for Cisco 8111 testbed
+        raw: connection create p42 to p47
+        ignore_errors: yes
+      
+      delegate_to: STR-W2W-SONIC-03-A
```

Comment:
Do not hard code specific internal lab hostname here. It must be read from somewhere else.